src/Common/EventListener/ResponseSecurer.php

```diff
@@ -14,16 +14,38 @@ class ResponseSecurer
      */
     public function onKernelResponse(FilterResponseEvent $event): void
     {
+        $cacheControl = 'max-age=0, must-revalidate, private, no-cache';
```

Comment:
I prefer
```$cacheControl = [
'max-age=0',
'must-revalidate', 
'private', 
'no-cache',
];```, and where line 19, adds it to the array, and on line 23, it can be imploded

Reply:
changed in df776be9d